### PR TITLE
Fix  pvc cleanup issue

### PR DESCRIPTION
**Pull request by ak-px**
**What this PR does / why we need it**:
In after suite the PVC  list validation was done when the backup locations were present. This caused the expected nfs pvc like nfs-backupsync , nfs-checkbackupfilemissing still exits in the cluster and subsequently causing the validation to fail.

PVC list after the run is [[alertmanager-px-backup-alertmanager-db-alertmanager-px-backup-alertmanager-0 alertmanager-px-backup-alertmanager-db-alertmanager-px-backup-alertmanager-1 prometheus-px-backup-dashboard-prometheus-db-prometheus-px-backup-dashboard-prometheus-0 prometheus-px-backup-dashboard-prometheus-db-prometheus-px-backup-dashboard-prometheus-1 pvc-nfs-backupsync-595112d pvc-nfs-checkbackupfilemissing-595112d pxc-mongodb-data-pxc-backup-mongodb-0 pxc-mongodb-data-pxc-backup-mongodb-1 pxc-mongodb-data-pxc-backup-mongodb-2 pxcentral-keycloak-data-pxcentral-keycloak-postgresql-0 pxcentral-mysql-pvc theme-pxcentral-keycloak-0]]

18:47:55  PVC 'pvc-nfs-backupsync-595112d' is present after the run but not before the run

https://jenkins.pwx.dev.purestorage.com/job/Builds/job/px-backup/job/px-backup-l1-nfs/543/consoleFull

With the PR moved the validation code after cleaning up all backup location, so that expected pvc will be deleted before we do a validation.

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/5164/consoleFull

 PVC list after the run is [[alertmanager-px-backup-alertmanager-db-alertmanager-px-backup-alertmanager-0 alertmanager-px-backup-alertmanager-db-alertmanager-px-backup-alertmanager-1 prometheus-px-backup-dashboard-prometheus-db-prometheus-px-backup-dashboard-prometheus-0 prometheus-px-backup-dashboard-prometheus-db-prometheus-px-backup-dashboard-prometheus-1 pxc-mongodb-data-pxc-backup-mongodb-0 pxc-mongodb-data-pxc-backup-mongodb-1 pxc-mongodb-data-pxc-backup-mongodb-2 ****-keycloak-data-****-keycloak-postgresql-0 ****-mysql-pvc theme-****-keycloak-0]]

### CODE REVIEW

**Review by mkoppal-px — APPROVED** [review]
LGTM

File: tests/backup/backup_basic_test.go [hunk 1]
```diff
@@ -435,6 +422,19 @@ var _ = AfterSuite(func() {
...
+		log.FailOnError(err, "failed to get Px-Backup namespace")
+		PvcListAfterRun, err = GetPVCListForNamespace(pxBackupNamespace)
+		log.FailOnError(err, "failed to list PVCs after run")
+		log.Infof("PVC list after the run is [%s]", PvcListAfterRun)
+
+		// Verify PVC Cleanup on PX-Backup namespace
+		if err := ValidatePVCCleanup(PvcListBeforeRun, PvcListAfterRun); err != nil {
+			log.FailOnError(err, "PVC cleanup validation failed")
+		}
+		fmt.Println("PVC cleanup validation passed.")
```
Comment (kphalgun-px) [inline tests/backup/backup_basic_test.go:437]: How about log.Infof?

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**